website/src/content/docs/general/index.md

```diff
@@ -4,10 +4,17 @@ sidebar:
 title: Introduction
 ---
 
-This manual contains general information on working with TrueCharts Charts.
+## Introduction to TrueCharts
 
-:::tip
+TrueCharts is a project dedicated to maintaining and developing Helm charts, with a particular focus on creating a shared, reusable common-lib chart for the Kubernetes ecosystem.
 
-Please, always remember to check the content specific to the chart.
+## Core Focus: common-lib and Documentation
 
-:::
+The primary focus of the core team is the development of the common-lib chart and its comprehensive documentation. This library serves as the foundational element for all other truecharts charts, providing consistency and ease of use across all charts.
+
+## Community Contributions and Staff Support
+
+While the majority of charts and their documentation are created and maintained by the community, they are continually supported by TrueCharts staff to ensure quality, reliability, and accuracy. The collaborative efforts between the community and staff are key to the success and growth of the project.
+
+## Collaborative Ecosystem
+TrueCharts fosters a community-driven environment where contributions are encouraged, enabling innovation and expanding the collection of Helm charts available for Kubernetes users.
```

Comment:
Great!

Feel free to adapt the pointers on the homepage accordingly if you like ^^
Love the collab ecosystem thingy!